data/models/reactions-model.js

```diff
@@ -0,0 +1,54 @@
+const db = require('../dbConfig');
+
+async function addReactions(reaction) {
+    try {
+        const ids = await db("reactions").insert(reaction, "id");
+        const id = ids[0];
+        const response = await findReactionsById(id);
+        return response;
+    } catch(error) {
+        console.log(error);
+    }
+}
+
+async function getReactions() {
+    try {
+        const reactions = await db("reactions as r")
+            .select(
+                'r.id', 
+                'highlighted_text', 
+                'emoji', 
+                'userId', 
+                'r.authorId', 
+                'r.title', 
+                'articleId',
+                'r.avatarUrl'
+            )
```

Comment:
Don't forget to give the relevant user names (if it's the reactor or author Id) after you have refactored your code based on the suggestion I made above.